src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,82 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (pageNumber: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pageAmount = Math.ceil(total / perPage);
+  const pageNumbers = [];
+
+  for (let i = 1; i <= pageAmount; i += 1) {
+    pageNumbers.push(i);
+  }
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames('page-item', { disabled: currentPage === 1 })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled="true"
+          onClick={() => {
+            if (currentPage > 1) {
+              onPageChange(currentPage - 1);
+            }
+          }}
```

Comment:
I think it's better to move this code to the function, and for the next button too.